lib/core/lib/ops/validation.js

```diff
@@ -4,14 +4,39 @@ const Joi = require('joi')
 
 const requiredString = Joi.string().required()
 const requiredStringId = Joi.string().required().max(128)
+const MetaData = Joi.object().optional().description('Arbitrary Metadata').label('MetaData')
 
+const PolicyIdObject = Joi.object({
+  id: Joi.string().required().description('Policy Id'),
```

Comment:
from what i saw the IDs are of type `requiredStringId` that has a max length limitation
Joi.string().required().description('Policy Id') -> requiredStringId.description('PolicyId')

Reply:
i'm gonna create PolicyId as a requiredId String and reuse that within the PolicyObject too 

I'm going to raise a change request for updating our models to share between validation.js and swagger.js